chart/values.yaml

```
@@ -14,6 +14,7 @@ prometheus:
   namespace: "monitoring"                    # K8s namespace where prometheus is installed
   namespace_am: "monitoring"                 # K8s namespace where prometheus-alertmanager is installed
   endpoint: "http://prometheus-server.monitoring.svc.cluster.local:80"   # HTTP Endpoint for Prometheus
+  scrapeInterval: 5
```

if this is seconds the key name or value should reflect that, so either:
```suggestion
  scrapeIntervalSeconds: 5
```
or

```suggestion
  scrapeInterval: "5s"
```
if we go for the second option we can probably support anything already supported by [time.ParseDuration](https://pkg.go.dev/time#ParseDuration)

---

@christian-kreuzberger-dtx Any preferences?

---

I implemented the second option since using the `time.duration` type directly seems like a better idea for larger time intervals.